### ISSUE TITLE
client names  not showing in dashboards

### ISSUE THREAD

**Issue by thefunkygibbon** [JGeek00/adguard-home-manager#155]
### Steps to reproduce

open eyes.
view dashboards
see up addresses
be sad

### Expected results

see names of devices

### Actual results

ip addresses

### Screenshots or Video

Screenshots or video demonstration
![Screenshot_20241224_230353_Chrome](https://github.com/user-attachments/assets/26624873-e410-4e93-8840-6a7ba1183c84)
![Screenshot_20241224_230337](https://github.com/user-attachments/assets/5988341a-9d55-41a2-967e-1ad530466d5c)
one screen shot is from the app ..the other is from the web interface.   as you can see, one is working as expected..the other, not so much

### App and OS details

latest as of today

android 14 Samsung s23 ultra

**Comment by JGeek00** [comment]
That issue is not happening for me.

![Screenshot_20250120-211235.png](https://github.com/user-attachments/assets/ecd89b8a-c9fb-48e7-b5b3-365edc8399ab)

![Screenshot_20250120-211226.png](https://github.com/user-attachments/assets/616845e4-18ee-4bb8-8def-7a75d54fbf97)

Make sure to have a name set there.
![Screenshot_20250120-211629.png](https://github.com/user-attachments/assets/13761638-5d4c-4461-8669-b98efa843c46)

**Comment by JGeek00** [comment]
Are you using AdGuard Home's DNS server? In my case the device name doesn't appear on the web app if I don't create a client.

**Comment by thefunkygibbon** [comment]
> Are you using AdGuard Home's DNS server? In my case the device name doesn't appear on the web app if I don't create a client.

well that's weird.   yes I'm using Adguard as my DHCP and DNS server.  I'm going to go out on a limb and assume that you are not using your Adguard home as a DHCP server on your network as well as DNS?  because that makes a difference I believe as the host is created (within the platform, but not as a "manually created client" when the DHCP lease is given.   

but that still leads to the question of what exactly is the process your app is using to display the list of hosts?  it must be doing something different to the normal web GUI 


<img width="582" alt="Image" src="https://github.com/user-attachments/assets/ec8752d0-f125-44fd-a2e8-4ed0f9b4ca73" />

<img width="498" alt="Image" src="https://github.com/user-attachments/assets/1f0af646-1415-48a2-bdcd-663c277c8320" />

**Comment by JGeek00** [comment]
@thefunkygibbon yes, I'm not using AdGuard Home's DHCP server. I use the DHCP server of my router. I think that names only appear when you use AdGuard Home's DHCP server. Since I don't use it, I don't have any way to test it. Probably on the app I'm printing a different parameter than the one that is used on the web app.
